tests/admin_filters/models.py

```diff
@@ -20,10 +20,13 @@ class Book(models.Model):
         related_name='books_authored',
         blank=True, null=True,
     )
-    contributors = models.ManyToManyField(User, verbose_name="Verbose Contributors", related_name='books_contributed', blank=True)
+    contributors = models.ManyToManyField(
+        User, verbose_name="Verbose Contributors", related_name='books_contributed', blank=True
```

Comment:
I'd suggest 1 option per line as done in the previous line. Please include a trailing comma after the last option to ease adding more.